pkg/scalers/kafka_scaler.go

```diff
@@ -177,12 +181,25 @@ func getKafkaClients(metadata kafkaMetadata) (sarama.Client, sarama.ClusterAdmin
 	config := sarama.NewConfig()
 	config.Version = sarama.V1_0_0_0
 
-	if ok := metadata.authMode == kafkaAuthModeForSaslPlaintext || metadata.authMode == kafkaAuthModeForSaslSSL || metadata.authMode == kafkaAuthModeForSaslScramSha256 || metadata.authMode == kafkaAuthModeForSaslScramSha512; ok {
+	if ok := metadata.authMode == kafkaAuthModeForSaslPlaintext || metadata.authMode == kafkaAuthModeForSaslSSL || metadata.authMode == kafkaAuthModeForSaslSSLPlain || metadata.authMode == kafkaAuthModeForSaslScramSha256 || metadata.authMode == kafkaAuthModeForSaslScramSha512; ok {
 		config.Net.SASL.Enable = true
 		config.Net.SASL.User = metadata.username
 		config.Net.SASL.Password = metadata.password
 	}
 
+	if metadata.authMode == kafkaAuthModeForSaslSSLPlain {
+		config.Net.SASL.Mechanism = sarama.SASLMechanism(sarama.SASLTypePlaintext)
+
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
Is this needed? I'd imagine eventhubs must have a valid cert, no?

Reply:
This is the same configuration as recommended by the Event Hubs team in the sample GitHub repo here - https://github.com/Azure/azure-event-hubs-for-kafka/blob/master/quickstart/go-sarama-client/consumer/sarama-consumer.go#L89
The authentication is done using the connection string being passed in the `username` and `password` metadata